koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -1029,11 +1029,22 @@ def get_source_build_info(self, build_id, build_nvr):
                 )
             raise koji.BuildError(err_msg)
 
+        build_type = self.session.getBuildType(koji_build['build_id'], strict=True)
+        if 'image' not in build_type:
+            type_string = list(build_type.keys())[0]
+            err_msg = ('koji build {} is {} build, source container '
+                       'build needs image build'.format(koji_build['nvr'], type_string))
+            raise koji.BuildError(err_msg)
+        elif koji_build['name'].endswith('-source'):
```

Comment:
Same comment as in https://github.com/containerbuildsystem/atomic-reactor/pull/1317#discussion_r352060822 -- I think we should look for a source_for_nvr key to decide whether it is a a source container image build.

Reply:
yes will change